modules/metas/internal/key/metaID.go

```diff
@@ -8,24 +8,33 @@ package key
 import (
 	"bytes"
 	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/persistenceOne/persistenceSDK/constants"
 	"github.com/persistenceOne/persistenceSDK/modules/metas/internal/module"
 	"github.com/persistenceOne/persistenceSDK/schema/helpers"
 	"github.com/persistenceOne/persistenceSDK/schema/types"
 	codecUtilities "github.com/persistenceOne/persistenceSDK/utilities/codec"
+	"strings"
 )
 
 type metaID struct {
+	TypeID types.ID `json:"typeID" valid:"required~required field typeID missing"`
 	HashID types.ID `json:"hashID" valid:"required~required field hashID missing"`
 }
 
 var _ types.ID = (*metaID)(nil)
 var _ helpers.Key = (*metaID)(nil)
 
 func (MetaID metaID) Bytes() []byte {
-	return MetaID.HashID.Bytes()
+	var Bytes []byte
+	Bytes = append(Bytes, MetaID.TypeID.Bytes()...)
+	Bytes = append(Bytes, MetaID.HashID.Bytes()...)
+	return Bytes
 }
 func (MetaID metaID) String() string {
-	return MetaID.HashID.String()
+	var values []string
+	values = append(values, MetaID.TypeID.String())
+	values = append(values, MetaID.HashID.String())
+	return strings.Join(values, constants.FirstOrderCompositeIDSeparator)
```

Comment:
we are using FirstOrderCompositeIDSeparator or DataTypeAndValueSeparator variable?